src/components/DevelopmentGuide/V3MigrationGuide.mdx

```diff
@@ -0,0 +1,155 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Button from "../Button/Button";
+import Link from "../Link/Link";
+import Text from "../Text/Text";
+
+<Meta title="Development Guide/v3 Migration Guide" />
+
+# Design System v3 Migration Guide
+
+## Table of Contents
+
+- {<Link href="#general-updates-requirements">General Updates & Requirements</Link>}
+- {<Link href="#upgrade-path">Upgrade Path</Link>}
+- {<Link href="#chakra-ui-2.8">Chakra UI 2.8</Link>}
+
+## General Updates & Requirements
+
+The Design System (DS) v3 release includes the following main updates:
+
+- Node 18+
+- React 18
+
+With these updates, the DS will offer better support for apps that are using the
+latest version of React. We strongly encourage this upgrade to take advantage of
+the latest features and improvements for React, Next.js, and other modern web
+development tools.
+
+Related to these updates, it is important to consider the following:
+
+- The Chakra UI 1.8 version is not supported by React 18.
+- Node 16 support ended in September, 2023 so while AWS and Vercel still
+  support it, they urgently recommend upgrading to a higher version.
+
+## Upgrade Path
+
+Not everything in this migration guide is applicable to all consuming
+applications. For example, even though the `Menu` component is a new addition,
+that does not mean that an application has to immediately adopt and use it.
+The ideal approach is to go through the "Breaking Changes” and “Additional
+Testing Needs” sections and update accordingly.
+
+### Main Additions and Upgrades
+
+- The `Menu` (new) and `MultiSelect` (v2) components are included in this release.
+- Chakra 1.8 does not support React 18, so Chakra has been updated to version
+  2.8. **IMPORTANT: If you manually and separately install Chakra in your app,
+  make sure to also upgrade it in the `package.json` file.**
+
+General updates can be found in the [Changelog](https://github.com/NYPL/nypl-design-system/blob/development/CHANGELOG.md)
+but main points to note:
+
+- `StatusBadge` has a new `type` prop.
+- There are better responsive styles for the `Heading` and `Text` components.
+- The following Chakra 2.8 helpers are now exported through the DS:
+  `cookieStorageManagerSSR`, `FocusLock`, `useStyleConfig`, and `useMultiStyleConfig`.
+
+### Major Visual Changes
+
+- `StatusBadge` has updated UI variants.
+- `Tabs` adds horizontal scrolling for desktop viewports and improves
+  horizontal scrolling for mobile viewports.
+
+## Breaking Changes
+
+### Design System
+
+- The `NewsletterSignup` and `SocialMediaLinks` components have been removed.
+  They will be added back in a later version.
+- `SimpleGrid` gap values have been updated to include `"grid"` in the prop
+  value name. For example, `gap=”grid.xs”`.
```

Comment:
I'm still not sure what to do about this.  We do use the `grid.*` values internally in a few components, but I am not sure why we decided to use this pattern.  The `GridGaps` type is also used for spacing in the `Form` component.  Was there some weird conflict?

Reply:
I just encountered this in Research Catalog with v2.1.6 actually. It's odd this did not occurred before. Here are more concrete examples using the v3 `rc`:

- https://github.com/NYPL/research-catalog/pull/115/files#diff-456863484b0393d53a579ae78a6e99086e7daa6fd0afdcc0c50e8fec2a0471eeR49
- https://github.com/NYPL/nypl-ds-test-app/pull/94/files#diff-a333ac33ea441affec39a9410d57a5d8c0e5d07e8e8bf3f1a4f9acb5900a0be7R59

Reply:
After moving to DS v3, did those apps start throwing errors due to the prop value?  Is that why they added the `grid` prefix?

Reply:
The Research Catalog threw this error in 2.1.6, so not necessarily a v3 issue.